daemon-tests/tests/liquidation.rs

```diff
@@ -0,0 +1,161 @@
+use daemon::projection::CfdState;
+use daemon_tests::confirm;
+use daemon_tests::expire;
+use daemon_tests::flow::next_with;
+use daemon_tests::flow::one_cfd_with_state;
+use daemon_tests::maia::OliviaData;
+use daemon_tests::mock_oracle_announcements;
+use daemon_tests::open_cfd;
+use daemon_tests::simulate_attestation;
+use daemon_tests::start_both;
+use daemon_tests::wait_next_state;
+use daemon_tests::OpenCfdArgs;
+use model::Dlc;
+use model::Price;
+use otel_tests::otel_test;
+use rand::prelude::SliceRandom;
+use rand::thread_rng;
+use rust_decimal::Decimal;
+
+#[otel_test]
+async fn given_open_cfd_when_oracle_attests_long_liquidation_price_can_liquidate() {
+    let (mut maker, mut taker) = start_both().await;
+
+    let oracle_data = OliviaData::example_0();
+
+    // We set the initial price to a value much higher than that of the price that the mock oracle
```

Comment:
Nifty! Took me a bit to understand the comment but makes total sense :)

Reply:
It's quite hard to explain :( Maybe there's a different way to write this code so that it doesn't require such a complex comment.

Reply:
I thought about it when I had a look, but did not have a better idea so far 😅
I think it's fine :)